modules/nw-openstack-external-ccm.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// tbd
+// TODO: GitHub link clearance.
+
+:_content-type: CONCEPT
+[id="nw-openstack-external-ccm_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM)
+
+In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy {rh-openstack} cloud provider to the external link:https://github.com/kubernetes/cloud-provider-openstack[OpenStack CCM]. This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
+
+To preserve user-defined configurations for the legacy cloud provider, existing configurations are mapped to new ones as part of the migration process. It searches for configurations in the following places:
+
+* `kube-cloud-config` in the `openshift-config-managed` namespace
+
+* `cloud-provider-config` in the `openshift-config` namespace
+
+Configurations that are found are synchronized to the `cloud-conf` config map in the `openshift-cloud-controller-manager` namespace.
```

Comment:
Again, `openshift-cloud-controller-manager` is merely a default. You can change this by passing the `--namespace` flag to the operator binary, as seen [here](https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/0acfa2d24deb6a03abce343608cfc6154650fd52/cmd/cluster-cloud-controller-manager-operator/main.go#L85-L89). I don't know if anyone can override this and [the default manifests do not](https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/0acfa2d24deb6a03abce343608cfc6154650fd52/manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml#L41-L49).